endpoints.go

```diff
@@ -120,6 +120,9 @@ var (
 	EndpointChannelWebhooks = func(cID string) string { return EndpointChannel(cID) + "/webhooks" }
 	EndpointWebhook         = func(wID string) string { return EndpointWebhooks + wID }
 	EndpointWebhookToken    = func(wID, token string) string { return EndpointWebhooks + wID + "/" + token }
+	EndpointWebhookMessage  = func(wID, token, messageID string) string {
+		return EndpointWebhookToken(wID, token) + "/" + "messages/" + messageID
```

Comment:
Why not just `"/messages/"` instead of separating it like that?

Reply:
pl fix this